pkgs/development/libraries/png++/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, libpng
+, docSupport ? true, doxygen ? null
+, testsSupport ? true
```

Comment:
You can get rid of this and just use the "doCheck" support below. The "checkPhase" will run "make test" by default.